src/setfit/modeling.py

```diff
@@ -280,25 +281,29 @@ def fit(
         self,
         x_train: List[str],
         y_train: Union[List[int], List[List[int]]],
-        num_epochs: int,
-        batch_size: Optional[int] = None,
-        learning_rate: Optional[float] = None,
-        body_learning_rate: Optional[float] = None,
+        classifier_num_epochs: int,
+        classifier_batch_size: Optional[int] = None,
+        body_classifier_learning_rate: Optional[float] = None,
```

Comment:
That would make more sense. The reason I went for `body_classifier_learning_rate` is because it allows for [this](https://github.com/tomaarsen/setfit/blob/dee70b151e46932049739acf59dd9a80ec45a85e/src/setfit/trainer.py#L327).
This unpacking sends the variable from [here](https://github.com/tomaarsen/setfit/blob/dee70b151e46932049739acf59dd9a80ec45a85e/src/setfit/training_args.py#L27-L32) to the fit method.

I can go either way with this change, changing the variable requires for changing the way the `fit` is called (but maybe explicitly calling the `fit` with specific arguments clearly is more readable than just sending all training arguments into the `fit` method).

Alright, I talked myself into changing this, haha.